PandaGrom/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  x = str.length / 2
+  str.length.odd? ? str[x] : str[x - 1] + str[x]
+end
+
+def run_cli
+  loop do
+    print 'Enter string: '
+    str = gets.chomp
+
+    break if str == 'exit!'
+
+    if str.empty?
+      p 'String can not be blank!'
```

Comment:
You used `print` in line 11 and now you are using `p`
Use the same method here to make your code consistent